modules/contracts/src.ts/constants.ts

```diff
@@ -0,0 +1,59 @@
+import { Options } from "yargs";
+
+export const botMnemonics = [
```

Comment:
I'm not sure we're even using these mnemonics anymore, we should just remove them.